src/mlpack/core/cv/metrics/topk_accuracy_impl.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file topk_accuracy_impl.hpp
+ * @author Arunav Shandeelya
+ * 
+ * The implementation of class topK_accuracy_score
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+#define MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+
+namespace mlpack {
+namespace cv {
+    template<typename MLAlgorithm, typename DataType, typename ResponseType, typename topK_score>
+    double topK_accuracy_score::Evaluate(MLAlgorithm& model,
+                                    const DataType& data,
+                                    const arma::Row<size_t>& labels,
+                                    const int TopK_Score& k)
+
+    {
+        if(data.n_cols! = labels.n_cols)
+        {
+            std::ostringstream fp;
+            fp << "topK_accuracy_score::Evaluate(): number of inputs ("data.n_cols")" 
+            << "does not matches with number of target labels ("labels.n_cols")" <<std::endl;
+        throw std::invalid_argument(fp.str());
+        }
+        arma::Row<size_t> predictedLabels;
+        // Taking Classification output from the model.
+        model.Classify(data, predictedLabels)
+        // shape of the predicted values
```

Comment:
Could you add a full stop at the end. Thanks a lot.

Reply:
Also these lines should start with a capital letter.

Reply:
Yeah, you are right. made the changes.